### DISCUSSION TITLE
New Twilio Navigation UI Kit Questions

### DISCUSSION THREAD

**Discussion by tonge495** [twilio-labs/paste#3228] [Q&A] [answered]
Hi,

I am trying out thew Twilio Navigation UI kit and had a few quick questions:
- Is the home icon on the top left navigation meant to go to the Console home page or is this something that is editable?
- Is the default top right section on the topbar meant for navigation around the accounts/subaccounts? What should display if it’s an account vs a sub account?
- Is the sidebar supposed to be collapsable/expandable or do we only use the default/compact options?

<img width="749" alt="Screenshot 2023-05-12 at 12 11 03 PM" src="https://github.com/twilio-labs/paste/assets/9394413/92fddb95-43ea-4a26-a2d7-ba92971f0a6b">

**Comment by loreina** [comment]
Hi @tonge495,

We're currently working on specs for the Navigation components and expect to have drafts up for input in the next couple of weeks. Those specs should have the answers you're looking for, though here are some cursory answers to unblock you in the meantime:

> Is the home icon on the top left navigation meant to go to the Console home page or is this something that is editable?

The home button in the Sidebar Header should route to the home page of that product/app.

> Is the default top right section on the topbar meant for navigation around the accounts/subaccounts? What should display if it’s an account vs a sub account?

Are you referring to the left or right of the topbar? The right actions are recommended to be reserved for application-level profile-related controls such as the search input, support button, product switcher, and user dialog.

> Is the sidebar supposed to be collapsable/expandable or do we only use the default/compact options?

The sidebar has collapsible functionality to account for Twilio products that currently have that type of navigation today. In general, it's up to your team or the product team to decide whether the product's sidebar should be collapsible.

**Reply by tonge495** [reply, marked as answer]
> Hi @tonge495,
> 
> We're currently working on specs for the Navigation components and expect to have drafts up for input in the next couple of weeks. Those specs should have the answers you're looking for, though here are some cursory answers to unblock you in the meantime:
> 
> > Is the home icon on the top left navigation meant to go to the Console home page or is this something that is editable?
> 
> The home button in the Sidebar Header should route to the home page of that product/app.
> 
> > Is the default top right section on the topbar meant for navigation around the accounts/subaccounts? What should display if it’s an account vs a sub account?
> 
> Are you referring to the left or right of the topbar? The right actions are recommended to be reserved for application-level profile-related controls such as the search input, support button, product switcher, and user dialog.
> 
> > Is the sidebar supposed to be collapsable/expandable or do we only use the default/compact options?
> 
> The sidebar has collapsible functionality to account for Twilio products that currently have that type of navigation today. In general, it's up to your team or the product team to decide whether the product's sidebar should be collapsible.

Thank you @loreina!  For my 2nd question, I was referring to the left of the topbar (where it says Account name/subaccount).

**Reply by loreina** [reply]
Hi @tonge495, the Account Switcher component is recommended to be placed there for users to see the current account they're on, and on trigger, it would open a menu to view other accounts the user has. The actual contents of that menu and any badge/button next to it would depend on the product. That guidance would come from those product teams. Are you asking for a specific product or in general?

**Reply by tonge495** [reply]
Just in general.  You answered my question here, thanks!
